src/main/scala/pipelines/text/AmazonReviewsPipeline.scala

```diff
@@ -71,11 +72,11 @@ object AmazonReviewsPipeline extends Logging {
     val conf = new SparkConf().setAppName(appName)
     conf.setIfMissing("spark.master", "local[2]") // This is a fallback if things aren't set via spark submit.
 
-    val sc = new SparkContext(conf)
+    val spark = SparkSession.builder.config(conf).getOrCreate()
```

Comment:
I'm only doing this here because it was the minimal set of changes required to get the Amazon pipeline working and everything to compile - SparkSQL .read.json requires some implicits to be in scope that come from importing implicits embedded in SparkSession object. Is use of SparkContext now deprecated?